internal/schemautil/mutations.go

```diff
@@ -73,5 +120,5 @@ func normalizeIpFilter(old, new map[string]interface{}) {
 		}
 	}
 
-	new["ip_filter"] = append(normalizedIpFilters, nonexistentIpFilters...)
+	new[fieldToWrite] = append(normalizedIpFilters, nonexistentIpFilters...)
```

Comment:
Please provide tests for this, with a good coverage.

Reply:
done